hawkbit-ui/src/main/java/org/eclipse/hawkbit/ui/common/table/BaseUIEntityEvent.java

```diff
@@ -41,4 +79,27 @@ public BaseEntityEventType getEventType() {
         return eventType;
     }
 
+    /**
+     * Check if the remove event is the same this ui event. Then maybe you can
+     * skip the remote event because is already executed.
+     * 
+     * @param tenantAwareEvent
+     *            the remote event
+     * @return <true> match <false> not match
+     */
+    public boolean matchRemoteEvent(final TenantAwareEvent tenantAwareEvent) {
+        if (!(tenantAwareEvent instanceof RemoteIdEvent) || entityClass == null || entityIds == null) {
+            return false;
+        }
+        final RemoteIdEvent remoteIdEvent = (RemoteIdEvent) tenantAwareEvent;
+        try {
+            final Class<?> remoteEntityClass = ClassUtils.getClass(remoteIdEvent.getEntityClass());
+            return entityClass.isAssignableFrom(remoteEntityClass) && entityIds.contains(remoteIdEvent.getEntityId());
+        } catch (final ClassNotFoundException e) {
+            LOG.error("Class not found", e);
```

Comment:
Please provide a more meaninful context about the error. Otherwise later on nowbody can figure out based on the log if this is a serious problem or not.